index.html

```diff
@@ -21,7 +21,7 @@
       <div class="row padding">
         <div class="column-full">
           <h1 class="inline">Code Journal</h1>
-          <div class="tab inline" data-view="entries">Entries</div>
+          <div class="tab inline" data-view="entry-form">Entries</div>
```

Comment:
This `div` shows the entries list, so it should be `data-view="entry-form"`. Also, it should be an `a` element, it is responsible for "navigation" between views.